app/src/main/kotlin/com/bitwarden/authenticator/ui/authenticator/feature/itemlisting/ItemListingViewModel.kt

```diff
@@ -403,101 +411,87 @@ class ItemListingViewModel @Inject constructor(
+                    is ItemListingState.ViewState.Content -> it.viewState.copy(
+                        actionCard = ItemListingState.ActionCardState.None,
+                    )
 
-    private fun authenticatorNoNetworkReceive(
-        state: DataState.NoNetwork<List<VerificationCodeItem>>,
-    ) {
-        if (state.data != null) {
-            updateStateWithVerificationCodeItems(
-                authenticatorData = state.data,
-                clearDialogState = true,
+                    is ItemListingState.ViewState.NoItems -> it.viewState.copy(
+                        actionCard = ItemListingState.ActionCardState.None,
+                    )
+                },
             )
-        } else {
-            mutableStateFlow.update {
-                it.copy(
-                    viewState = ItemListingState.ViewState.Error(
-                        message = R.string.internet_connection_required_title
-                            .asText()
-                            .concat(R.string.internet_connection_required_message.asText()),
-                    ),
-                    dialog = null,
-                )
-            }
         }
     }
 
-    private fun authenticatorPendingReceive(
-        action: DataState.Pending<List<VerificationCodeItem>>,
-    ) {
-        updateStateWithVerificationCodeItems(
-            authenticatorData = action.data,
-            clearDialogState = false,
-        )
-    }
+    /**
+     * Converts a [SharedVerificationCodesState] into an action card for display.
+     */
+    private fun SharedVerificationCodesState.toActionCard(): ItemListingState.ActionCardState =
+        when (this) {
+            SharedVerificationCodesState.AppNotInstalled ->
+                if (!settingsRepository.hasUserDismissedDownloadBitwardenCard) {
+                    ItemListingState.ActionCardState.DownloadBitwardenApp
+                } else {
+                    ItemListingState.ActionCardState.None
+                }
 
-    private fun updateStateWithVerificationCodeItems(
-        authenticatorData: List<VerificationCodeItem>,
-        clearDialogState: Boolean,
-    ) {
-        mutableStateFlow.update { currentState ->
-            currentState.copy(
-                viewState = authenticatorData.toViewState(
-                    alertThresholdSeconds = state.alertThresholdSeconds,
-                ),
-                dialog = currentState.dialog.takeUnless { clearDialogState },
-            )
+            SharedVerificationCodesState.SyncNotEnabled ->
+                // TODO: Implement sync not enabled action card
+                ItemListingState.ActionCardState.None
+
+            SharedVerificationCodesState.Error,
+            SharedVerificationCodesState.FeatureNotEnabled,
+            SharedVerificationCodesState.Loading,
+            SharedVerificationCodesState.OsVersionNotSupported,
+            is SharedVerificationCodesState.Success,
+                -> ItemListingState.ActionCardState.None
```

Comment:
⛏️ This is odd. In PM `->` is not indented. 🤔

Reply:
This must be due to a recent change in the AS formatter settings. It's happening in the PM app now, too. Instead of fighting the formatter we're going to keep the indentions. I'll whip up a PR to update all the instances in one shot.